src/tests/functional/shared_test_classes/include/shared_test_classes/base/utils/ranges.hpp

```diff
@@ -4,152 +4,383 @@
 #include "openvino/op/hard_sigmoid.hpp"
 #include "openvino/op/hsigmoid.hpp"
 #include "openvino/op/hswish.hpp"
+#include "openvino/op/idft.hpp"
+#include "openvino/op/if.hpp"
+#include "openvino/op/interpolate.hpp"
+#include "openvino/op/irdft.hpp"
 #include "openvino/op/log.hpp"
+#include "openvino/op/logical_and.hpp"
+#include "openvino/op/logical_not.hpp"
+#include "openvino/op/logical_or.hpp"
+#include "openvino/op/logical_xor.hpp"
+#include "openvino/op/loop.hpp"
+#include "openvino/op/lrn.hpp"
+#include "openvino/op/lstm_cell.hpp"
+#include "openvino/op/lstm_sequence.hpp"
+#include "openvino/op/matmul.hpp"
+#include "openvino/op/matrix_nms.hpp"
+#include "openvino/op/max_pool.hpp"
+#include "openvino/op/maximum.hpp"
+#include "openvino/op/minimum.hpp"
 #include "openvino/op/mish.hpp"
+#include "openvino/op/mod.hpp"
+#include "openvino/op/multinomial.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/mvn.hpp"
 #include "openvino/op/negative.hpp"
+#include "openvino/op/non_max_suppression.hpp"
+#include "openvino/op/pad.hpp"
 #include "openvino/op/parameter.hpp"
+#include "openvino/op/power.hpp"
 #include "openvino/op/prelu.hpp"
+#include "openvino/op/proposal.hpp"
+#include "openvino/op/psroi_pooling.hpp"
+#include "openvino/op/random_uniform.hpp"
+#include "openvino/op/range.hpp"
+#include "openvino/op/rdft.hpp"
+#include "openvino/op/reduce_l1.hpp"
+#include "openvino/op/reduce_l2.hpp"
+#include "openvino/op/reduce_logical_and.hpp"
+#include "openvino/op/reduce_logical_or.hpp"
+#include "openvino/op/reduce_max.hpp"
+#include "openvino/op/reduce_mean.hpp"
+#include "openvino/op/reduce_min.hpp"
+#include "openvino/op/reduce_prod.hpp"
+#include "openvino/op/reduce_sum.hpp"
+#include "openvino/op/region_yolo.hpp"
 #include "openvino/op/relu.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/reverse_sequence.hpp"
+#include "openvino/op/rnn_sequence.hpp"
+#include "openvino/op/roi_align.hpp"
+#include "openvino/op/roi_pooling.hpp"
+#include "openvino/op/roll.hpp"
 #include "openvino/op/round.hpp"
+#include "openvino/op/scaled_dot_product_attention.hpp"
+#include "openvino/op/scatter_elements_update.hpp"
+#include "openvino/op/scatter_update.hpp"
+#include "openvino/op/select.hpp"
 #include "openvino/op/selu.hpp"
+#include "openvino/op/shuffle_channels.hpp"
 #include "openvino/op/sigmoid.hpp"
 #include "openvino/op/sign.hpp"
 #include "openvino/op/sin.hpp"
 #include "openvino/op/sinh.hpp"
+#include "openvino/op/slice.hpp"
+#include "openvino/op/softmax.hpp"
 #include "openvino/op/softplus.hpp"
 #include "openvino/op/softsign.hpp"
+#include "openvino/op/space_to_batch.hpp"
 #include "openvino/op/sqrt.hpp"
+#include "openvino/op/squared_difference.hpp"
+#include "openvino/op/strided_slice.hpp"
+#include "openvino/op/subtract.hpp"
 #include "openvino/op/swish.hpp"
 #include "openvino/op/tan.hpp"
 #include "openvino/op/tanh.hpp"
-#include "openvino/op/max_pool.hpp"
+#include "openvino/op/tensor_iterator.hpp"
```

Comment:
Minor: `#include "openvino/ops.hpp"` instead of all ops?

Reply:
done